docs/src/intervention.html.jinja2

```diff
@@ -590,13 +618,26 @@
                     </div>
 
                     <div class="row">
-                        <div class="col-md-6">
+                        <div class="col-md-2">
+                            <label>Discharged with home O₂?</label>
```

Comment:
Consider using <subscript> here. Not sure how well characters like this will translate through M Web Server or Apache, etc. IMO, the safer bet is to have the browser render it using a subscript tag.

Reply:
Yeah, I'll do that.

Reply:
Oh. Right. I'd forgotten—I did use `<sup>` the first time, but the `build.sh`script formats the HTML like so:

```
        <label>
         Discharged with home O₂?
         Discharged with home O
         <sub>
          2
         </sub>
         ?
        </label>
```

So, there are additional spaces around the subscripted 2. Nevertheless, I've let this stand as is, though the typographer deep within me is glaring pretty hard at that.

Reply:
Here's a different alternative. Just use the unicode HTML equivalent: `&#8322;`